pkgs/by-name/fu/fuzzel/package.nix

```diff
@@ -54,14 +55,15 @@ stdenv.mkDerivation (finalAttrs: {
     tllist
     fcft
   ] ++ lib.optional enableCairo cairo
-    ++ lib.optional pngSupport libpng;
+  ++ lib.optional pngSupport libpng
+  ++ lib.optional (svgBackend == "librsvg") librsvg;
 
   mesonBuildType = "release";
 
   mesonFlags = [
     "-Denable-cairo=${if enableCairo then "enabled" else "disabled"}"
     "-Dpng-backend=${if pngSupport then "libpng" else "none"}"
-    "-Dsvg-backend=${if svgSupport then "nanosvg" else "none"}"
+    "-Dsvg-backend=${svgBackend}"
```

Comment:
Nitpick: we could also use `(lib.mesonFeature "svg-backend" svgBackend)` here (and with the other flags)

Reply:
> We may want to look into making this backwards compatible with the previous override

That's a good idea. I reworked the commit guided by your suggestions. I wasn't able to implement them exactly as you proposed but hope I was able to capture the intent.